tg-bot/src/main/java/com/tutrit/tgbot/example/repository/RepositoryUserEmployee.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.tgbot.example.repository;
+
+import com.tutrit.tgbot.example.bean.UserEmployee;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class RepositoryUserEmployee {
+
+    public static Object[] arrayEmployee = new Object[1];
```

Comment:
ArrayType should be Engineer, 
I think it would be better to initialize it with 10 entries